gateway/1.9/modules/ROOT/pages/flex-gateway-secure-conn.adoc

```diff
@@ -1,13 +1,48 @@
-= Securing and Governing APIs in Connected Mode
+= Applying Policies for Managed Flex Gateways and Connected Mode
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-For Flex Gateway running in Connected Mode, you apply API policies using *API Manager*. Refer to xref:policies-included-apply.adoc[] for more information.
+:page-aliases: policies-included-apply.adoc, api-manager::using-policies.adoc, policies::policies-included-apply.adoc
 
-Additionally, to apply a basic authentication policy, see xref:flex-gateway-getting-started.adoc[].
+For Managed Flex Gateway and Flex Gateway running in Connected Mode, you apply inbound and outbound policies using *API Manager*. To find the policies provided by Flex Gateway, see xref:policies-included-directory.adoc[] and xref:policies-outbound-directory.adoc[].
+
+== Apply an Inbound Policy
+
+. Go to *Anypoint Platform > API Manager*.
+. In *API Administration* click the name of the API to which to apply a policy.
+. From the left navigation menu, click *Policies*.
+. Click *+ Add policy*.
+. Select the policy to apply.
+. Configure the required configuration parameters.
++
+For policy configuration parameters, see xref:policies-included-directory.adoc[].
+. If necessary, configure *Advanced options*.
+. Click *Apply*
+
+You can apply a policy either to the entire API (default) or to specific methods and resources of an API.
+
+== Apply an Outbound Policy
+
+. Go to *Anypoint Platform > API Manager*.
+. In *API Administration* click the name of the API to which to apply a policy.
+. From the left navigation menu, click *Policies*.
+. Click the *Outbound policies* tab.
+. Click *+ Add outbound policy*.
+. Select the *Upstream* to apply a policy to.
```

Comment:
```suggestion
. Select the *Upstream* to which you want to apply a policy.
```

Reply:
You might check in [#cx-doc-style](https://salesforce.enterprise.slack.com/archives/C01U1MSQ85V) to see if it's okay to end a sentence with a preposition. It might be considered conversational. I didn't see anything in the style guidelines.